components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -29,76 +37,105 @@ export const contactSalesCard = (contactSalesLink: any, isFreeTrial: boolean, tr
             />
         );
     } else {
-        title = (
-            <FormattedMessage
-                id='admin.billing.subscription.privateCloudCard.cloudProfessional.title'
-                defaultMessage='Upgrade to Cloud Enterprise'
-            />
-        );
-        description = (
-            <FormattedMessage
-                id='admin.billing.subscription.privateCloudCard.cloudProfessional.description'
-                defaultMessage='Optimize your processes with VPC Peering, a dedicated AWS account and premium support.'
-            />
-        );
+        switch (subscriptionPlan) {
+        case 'cloud-starter':
```

Comment:
Make these `cloud-starter` `cloud-professional` etc constants. If we ever change them, you will have to update multiple switch statements, but with constants only change one variable.